src/main/java/com/github/wovnio/wovnjava/HtmlChecker.java

```diff
@@ -4,6 +4,10 @@
 import org.jsoup.nodes.Element;
 
 class HtmlChecker {
+    public boolean isTextFileContentType(String contentType) {
```

Comment:
I don't think there is a need for this. Just use `canTranslateContentType`. I think we should remove `canTranslate` and call the parts individually

Reply:
Or is it not possible for non-html text responses to go through the binary branch?

Reply:
There's a subtle difference between "translatable" and "non-binary". For example, CSS files are not translatable, but they might have `::before` stuff in Shift_JIS.